### PR TITLE
Login and welcome page

### LINKED ISSUE #33: Customer welcoming page
As a user I want to easily access other features as soon as I have logged in. -8 pts

Acceptance Criteria: Customer accesses welcoming page when logged in.

### PR DESCRIPTION
**Pull request by rinataka**
closes #33

### CODE REVIEW

File: src/DatabaseController.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ var conString = "postgres://tlifeerj:JxvQP_5LdXUAQd5reuIWuK2WbmlnB74B@hattie.db.
 var client = new pg.Client(conString);
```
Comment (DaddyFresh) [inline src/DatabaseController.js:6]: Looks like DatabaseContoler is not being used so romve this compleatley

File: src/Css/OrderInterface.css [hunk 2]
```diff
@@ -1,4 +1,5 @@
 .OrderInterface {
```
Comment (DaddyFresh) [inline src/Css/OrderInterface.css:1]: nice coloring changes, good job!

**Review by DaddyFresh — CHANGES_REQUESTED** [review]
To big pull request, in the future we should try and spreed out our requests

**Review by oodam — APPROVED** [review]
looks good but big

**Review by DaddyFresh — APPROVED** [review]

**Pull request merged**